.gitignore

```diff
@@ -65,3 +65,9 @@ coverage.xml
 !.gitignore
 .noseids
 Procfile
+
+# Local configuration file of udata
+udata.cfg
+
+# Local data directory
+data
```

Comment:
As far as I remember, the 'data' directory has been created by default when I installed the platform. And it is explicitly referenced in docker-compose.yml.

'udata.cfg' is not really created by default, but, if you do not specify UDATA_SETTINGS, it is the file that udata looks for. It is then the default user configuration file.

For me, it is reasonable to put them in the global .gitignore.